roles/setup_pgd/defaults/main.yml

```diff
@@ -113,3 +113,12 @@ supported_pg_version:
 
 supported_pgd_version:
   - 5
+
+# Other PG Users
+enterprisedb_user: "enterprisedb"
```

Comment:
Why is this listed? Doesn't the `/vars/EPAS_RedHat` cover the `pg_owner: enterprisedb` case?

Reply:
I agree, pg_owner should be used where you found a need to create enterprisedb_user.